Json reneder

Taar opened the pull request:
25c9dae
- Remove the os signal call out of the view
- Add a request method that injects a function into the request based on
  the environment. In development that function does nothing. In
  production it'll make the io signal call

4356aa0
- Update the default view to return a json payload
- Update the error file tween to return a json payload
- Add/Update tests

c907fe1
- Add tests for pid tween
- Fix an issue with the Spy not returning the right object
- Update pid tween to responsed with a json payload

Taar committed:
Making development easier

- Remove the os signal call out of the view
- Add a request method that injects a function into the request based on
  the environment. In development that function does nothing. In
  production it'll make the io signal call

Taar committed:
Convert Endpoints to return json

- Update the default view to return a json payload
- Update the error file tween to return a json payload
- Add/Update tests

Taar committed:
Tests for pid tween

- Add tests for pid tween
- Fix an issue with the Spy not returning the right object
- Update pid tween to responsed with a json payload

wcomartin commented:
:+1: looks good to me. 

although I am getting a `{"message":"PID file not found in file system! /tmp/door_controller.pid"}` error in development i think thats expected

Taar committed:
json_reneder

- Remove static files
- Remove templates files
- Add password reset post endpoint
